arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -0,0 +1,1707 @@
+          "type": "boolean"
+        },
+        "scope": {
+          "description": "The scope of repositories where the event can be triggered.",
+          "type": "string"
+        },
+        "actions": {
+          "description": "The list of actions that trigger the webhook to post notifications.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "WebhookListResult": {
+      "description": "The result of a request to list webhooks for a container registry.",
+      "properties": {
+        "value": {
+          "description": "The list of webhooks. Since this list may be incomplete, the nextLink field should be used to request the next list of webhooks.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Webhook"
+          }
+        },
+        "nextLink": {
+          "description": "The URI that can be used to request the next list of webhooks.",
+          "type": "string"
+        }
+      }
+    },
+    "EventListResult": {
+      "description": "The result of a request to list events for a webhook.",
+      "properties": {
+        "value": {
+          "description": "The list of events. Since this list may be incomplete, the nextLink field should be used to request the next list of events.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Event"
+          }
+        },
+        "nextLink": {
+          "description": "The URI that can be used to request the next list of events.",
+          "type": "string"
+        }
+      }
+    },
+    "Event": {
+      "description": "The event notification for a webhook.",
+      "properties": {
+        "id": {
+          "description": "ID provides a unique identifier for the event.",
+          "type": "string"
+        },
+        "timestamp": {
+          "description": "Timestamp is the time at which the event occurred.",
+          "type": "string"
+        },
+        "action": {
+          "description": "Action indicates what action encompasses the provided event.",
+          "type": "string"
+        },
+        "target": {
+          "$ref": "#/definitions/Target",
+          "description": "Target uniquely describes the target of the event."
+        },
+        "request": {
+          "$ref": "#/definitions/Request",
+          "description": "Request covers the request that generated the event."
+        },
+        "actor": {
+          "$ref": "#/definitions/Actor",
+          "description": "Actor specifies the agent that initiated the event. For most situations, this could be from the authorization context of the request."
+        },
+        "source": {
+          "$ref": "#/definitions/Source",
+          "description": "Source identifies the registry node that generated the event. Put differently, while the actor \"initiates\" the event, the source \"generates\" it."
+        }
+      }
+    },
+    "Target": {
```

Comment:
The entire thing under `EventListResult` will appear only in response. From the discussions in our previous swagger reviews, adding `readonly` would be redundant.